WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -28130,7 +28130,7 @@
 	culture = old_slavic
 }
 1042167 = {
-	name="Atillid" 
+	name="the Hun" 
 	culture = hunnic 
```

Comment:
It's still a terrible decision. "Timmy the Hun" as Ernakh's son, or installing "House the Hun" as your vassal if you can entice an Attila relative to come to your court is so much for fun than the generic "-id" suffix. It's also somewhat more historical because "the Hun" is not ocnjecture like "Attilid" is.